docs/UserGuide.md

```diff
@@ -221,6 +239,13 @@ Adds an event to TalentHub.
 
 Format: `add event n/NAME t/TIME [v/VENUE] c/CELEBRITY [p/POINTS OF CONTACT]…​`
 
+<div markdown="block" class="alert alert-info"> 
+
+**:information_source: Event constraints:**<br>
+
+- The celebrity and all points of contacts must be existing persons in TalentHub.
```

Comment:
Should we put person in codeblock to make it clear that its not a grammatical error